electrical/readme.MD

```diff
@@ -0,0 +1,261 @@
+-->
+# Open Source Rover: Electrical Assembly Instructions
+<!--Authors: Michael Cox, Eric Junkins, Olivia Lofaro
+
+Reference herein to any specific commercial product, process, or service by trade name, trademark, manufacturer, or otherwise, does not constitute or imply its endorsement by the United States Government or the Jet Propulsion Laboratory, California Institute of Technology.
+
+Copyright 2018 California Institute of Technology. Government sponsorship acknowledged
+-->
+# 1. Overview
+First, it is important to examine the components we will use in the rover and understand how both data and power travel through the system. Figure 1 shows how each of the electrical components and boards receives power, and Figure 2 shows how data are sent from each of the electrical systems and what communication protocol it uses.
+
+
+
+<img src="../images/electronics/power_diagram.png" width="100%">
+
+**Figure 1. Electrical Power Overview**
+
+One important thing to note from the above is that there are multiple different voltages in the system. There is the unregulated voltage coming from the battery, 5V regulated coming from the voltage regulator, and the 12V motor voltage which is sent from the motor controllers to the motors.
+
+<img src="../images/electronics/data_comm_diagram.png" width="100%">
+
+  **Figure 2. Electrical Data Overview**
+
+For the data transmission, we use both 3.3V and 5V digital logic systems, analog signals (of varying max voltages), and serial communication between devices. Information on these protocols can be found at:
+
+* [Serial Communications](https://en.wikipedia.org/wiki/Serial%20communication)
+* [Analog vs. Digital](https://learn.sparkfun.com/tutorials/analog-vs-digital)
+
+**Table 1. Electrical Components**
+
+Part | Description
+---- | -----------
+Battery | Provides power to the system. Has an unregulated voltage range of around 11.5V - 16.75V depending on its charge level
+Switch | Mechanical disconnect of the electrical power to the rest of the robot
+Volt Meter | Monitors and displays information such as current draw, voltage level, and power consumption of the robot
+Voltage Regulator | Takes in the unregulated battery voltage and outputs a steady regulated voltage for consumption by electronics.
+Raspberry Pi 3 | Acts as the brains and processing power of the robot
+Arduino Uno | Runs the LED Matrix
+LED Matrix | Gives the robot a display and a personality
+Motor Controllers | Sends voltage signals to the motors telling the motors the direction and speed at which to spin. Reads encoder data
+Encoders | Monitors the position of the motor to give information about speed and position
+Control Board PCB | Custom Printed Circuit Board to manage power and data transmition between electronics components.
+
+
+## 1.1. Tools Necessary
+
+**Table 2. Necessary Tools**
+
+Part | Image
+---- | -----
+Heat Shrink Gun |<p align="center"> <img src="../images/electronics/tools/heat_gun.png"> </p>
```

Comment:
Make this table 2 columns, refer to https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md

Reply:
done